src/dataexplorer/dataexplorer-application/src/components/Patient/PatientQuestionnaireResponses/PatientQuestionnaire/PatientQuestionnaire.tsx

```diff
@@ -4,6 +4,7 @@ import { IPatientIdentifier, IDataSource } from 'types/IPatient';
 import { IQuestionnaireResponse } from 'types/IQuestionnaireResponse';
 import PatientView from './PatientView/PatientView';
 import { Row, Spin } from 'antd';
+import '../../Patient-style.scss';
```

Comment:
I think it's prettier to import from src in this case
```suggestion
import 'components/Patients/Patient-style.scss';
```